packages/terra-functional-testing/README.md

```diff
@@ -0,0 +1,24 @@
+# Terra Functional Testing
+
+[![NPM version](https://badgen.net/npm/v/@cerner/terra-functional-testing)](https://www.npmjs.com/package/@cerner/terra-functional-testing)
+[![Build Status](https://badgen.net/travis/cerner/terra-toolkit)](https://travis-ci.com/cerner/terra-toolkit)
+
+A functional testing utility for applications built using Terra.
+
+- [Getting Started](#getting-started)
+- [Documentation](https://github.com/cerner/terra-toolkit/tree/master/packages/terra-functional-testing/README.md)
+- [LICENSE](#license)
+
+## Getting Started
+
+- Install from [npmjs](https://www.npmjs.com): `npm install --save-dev @cerner/terra-functional-testing`
+
+## LICENSE
+
+Copyright 2018-2020 Cerner Innovation, Inc.
```

Comment:
Are you pulling the original terra-toolkit years for this?

Reply:
I copy pasted this from another license, it looks like terra-toolkit started in 2017. Should new packages start in 2020?

Reply:
Yeah, start new packages in the year they were created.

Reply:
License years have been updated: https://github.com/cerner/terra-toolkit/pull/403/commits/805b033f99c127f51c3555129e68c07d72289f27